Dockerfile.exec

```diff
@@ -0,0 +1,22 @@
+FROM phusion/baseimage
+MAINTAINER Library Simplified <info@librarysimplified.org>
+
+ARG version
+ARG repo="NYPL-Simplified/circulation"
+
+ENV SIMPLIFIED_DB_TASK "auto"
+ENV SIMPLIFIED_SCRIPT_NAME ""
+
+# Copy over all Library Simplified build files for this image
+COPY . /ls_build
+
+RUN /bin/bash -c "/ls_build/simplified_app.sh ${repo} ${version} \
+      && /ls_build/logrotate.sh \
+      && rm -rf /ls_build && /bd_build/cleanup.sh"
+
+VOLUME /var/log
+WORKDIR /home/simplified/circulation/bin
+
+CMD ["/sbin/my_init", "--skip-runit", "--quiet", "--", \
```

Comment:
The command here depends on the documentation for the base image of this Dockerfile (phusion/baseimage): https://github.com/phusion/baseimage-docker#running-a-one-shot-command-in-a-new-container.